front_end/src/actions/AdGroups/AdGroupActions.js

```diff
@@ -1,11 +1,7 @@
 import fetch from 'isomorphic-fetch';
+import * as config from 'helpers/config';
 
-let apiUrl;
-if (typeof __DEVELOPMENT__ !== 'undefined' && __DEVELOPMENT__ === true) {
-  apiUrl = __DEVAPI__;
-} else {
-  apiUrl = __LIVEAPI__;
-}
+const apiUrl = config.get('DEVELOPMENT') ? config.get('DEV_API') : config.get('LIVE_API');
```

Comment:
Now that we have nicer settings. Should we look at just having one `API_URL` setting? Having only DEV_ and LIVE_ doesn't really help given we have local, dev, stage and prod environments.

Doesn't have to go in this PR.